tests/unit/scheduler.c

```diff
@@ -336,6 +337,27 @@ int test_threshold_scheduler()
 	}
 	nccl_net_ofi_release_schedule(scheduler, schedule);
 
+	schedule = scheduler->get_schedule(scheduler, rr_threshold + 1, num_rails);
+	if (!schedule) {
+		NCCL_OFI_WARN("Failed to get schedule");
+		free(ref_schedule);
+		return -1;
+	}
+	ref_schedule->num_xfer_infos = 2;
+	ref_schedule->rail_xfer_infos[0].rail_id = 2;
+	ref_schedule->rail_xfer_infos[0].offset = 0;
+	ref_schedule->rail_xfer_infos[0].msg_size = rr_threshold / 2 + 128;
```

Comment:
Where is 128 coming from? It must be right, since it was there before and the test is passing, but I am curious :)
It would be more readable if these numbers (`2` and `128`)  were in variables with meaningful names.